src/assets/_project/_blocks/data/qg-google-keys.json

```diff
@@ -1,6 +1,6 @@
 {
   "defGoogle" : {
-    "test" : "AIzaSyA1uwIi2C0x9VbCqoVK4nxcID4CVqF3uhQ",
+    "test" : "AIzaSyCKuaFIFo7YYZXHZ5zaiEZdJx0UBoyfuAE",
```

Comment:
did we document which google key this is connected on our wiki?

Reply:
AIzaSyCKuaFIFo7YYZXHZ5zaiEZdJx0UBoyfuAE is the test key, the other was not working.

What you mean by connecting on the wiki?

Reply:
I think he meant "document on the wiki", not "connected on the wiki".

Reply:
yes that

Reply:
I have added in the wiki https://servicesmadesimpler.govnet.qld.gov.au/wiki/display/QOL/Google+Keys+details